lte/gateway/python/magma/subscriberdb/client.py

```diff
@@ -123,7 +139,9 @@ async def _get_subscribers(self) -> [SubscriberData]:
         SUBSCRIBER_SYNC_LATENCY.observe(
             time_elapsed.total_seconds() * 1000,
         )
-        return subscribers
+
+        self._store.update_digest(cur_digest.md5_base64_digest)
+        return subscribers, no_updates
```

Comment:
Make sure we test what happens when cloud sends down empty digest

Reply:
Currently the logic works as such: when cloud sends down an empty digest, the gateway will store the empty gateway and send it in its next request; however, if `ListSubscribersRequest.Digest = ""`, then cloud will not enable the `no_updates` logic, but will send down the full subscriber list (as well as a new digest if it has one) as usual. Wondering if you think this is a reasonable logic? @hcgatewood

Reply:
Yes this is how it should work, makes sense. My note was just to verify that this is being tested